requirements.txt

```diff
@@ -1,5 +1,5 @@
-Django==3.1.13
-asgiref==3.2.10
```

Comment:
I have a small memory that something about the live attendance list at events broke at a certain version of asgiref & channels. We will probably have to test the websocket again and/or make changes to it. Or remove it or something :D

Reply:
Websocket does indeed not seem to be working. I will look into that, added to PR tasks.

Reply:
Websockets should now work, I added daphne as an additional requirement.